cfgov/wellbeing/jinja2/wellbeing/home.html

```diff
@@ -12,35 +12,16 @@
     {{ super() }}
     <style>
         .m-hero {
-            background-color: #cce3f5;
+            background-color: #b4d2d1;
         }
         .m-hero_image {
-            background-image: url( '/f/images/parents_hero_760x364.original.png' );
+            background-image: url( '{{ static("img/fwb_hero_760x300.png") }}' );
             filter: progid:DXImageTransform.Microsoft.AlphaImageLoader(
-                src='/f/images/parents_hero_760x364.original.png',
-                sizingMethod='scale'
+                    src='{{ static("img/fwb_hero_760x300.png") }}',
+                    sizingMethod='scale'
             );
             padding-bottom: 41.4893617%;
         }
-        @media screen and ( min-width: 37.5625em ) {
```

Comment:
The padding bottom controls the size of the image box, and by extension, the size of the image itself (which is a background image with `background-size: contain`). It's calculated when Wagtail outputs the page, so your comment made me realize that I needed to update it to get the correct intended size for this image. Fix pushed!